README.md

```diff
@@ -1,79 +1,93 @@
 # Distro plugin for [Tutor](https://docs.tutor.overhang.io)
 
-
-This plugin is a tool to facilitate the customization of an Openedx instance, adding commands and settings to have an easy-to-use
-and a ready-to-deploy in local or in development openedx distribution.
-
+This plugin is a tool to facilitate the customization of an Open edX instance, adding commands and settings to have an easy-to-use
+and a ready-to-deploy in a local or development environment.
 
 ## Installation
 
-To install the latest release
+To install the latest release, run:
 
 ```bash
 pip install git+https://github.com/eduNEXT/tutor-contrib-edunext-distro
 ```
 
-You can install a specific version by adding the tag at the end, e.g, `pip install git+https://github.com/eduNEXT/tutor-contrib-edunext-distro#v.17.0.0`
+You can install a specific version by adding the tag at the end, e.g, `@v17.0.0`
 
 ## Usage
 
+1. **Enable the plugin**: after installing the plugin, enable it by running:
+
 ```bash
 tutor plugins enable distro
+```
+
+2. **Configure the Distro plugin**: this plugin adds a new set of settings used to further customize your Open edX installation. Refer to the following documentation to know the Distro variables:
+
+- [How to customize distro](./docs/how_to_customize_distro.rst)
+- [How to add a new package](./docs/how_to_add_new_packages.rst)
 
-# Validator commands for config file
+> [!NOTE]
```

Comment:
This is cool! We should use it in all the repos.

Reply:
Unfortunately, it's only supported for markdown not rst.